golem-service-base/src/model.rs

```diff
@@ -2807,6 +2807,7 @@ pub struct WorkerMetadata {
     pub last_error: Option<String>,
     pub component_size: u64,
     pub total_linear_memory_size: u64,
+    pub owned_resources: HashMap<u64, ResourceMetadata>,
```

Comment:
did not this changed also OpenApi spec? (if it needs to be regenerated)

Reply:
yes it did, now need to update the cli because of it :) one more commit